C_CPP/Fundamentals/Static scoping & Dynamic Scoping/readme.md

```diff
@@ -1 +1 @@
-
+- Static and Dynamic scoping 
```

Comment:
```suggestion
- Static and Dynamic scoping (./Static scoping & Dynamic Scoping.md)
```

Reply:
@vishnu95075 Just copy and paste the above suggested index line, you have to add this relative link